cmd/repo/setup/command.go

```diff
@@ -0,0 +1,54 @@
+package setup
+
+import (
+	"io"
+	"os"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/cobra"
+)
+
+const (
+	name        = "setup"
+	description = "Configure github repository settings and permissions."
```

Comment:
The env var for `GITHUB_TOKEN` should be explained / mentioned here, so it turns up in the helptext.